src/components/combat/Combat.module.scss

```diff
@@ -0,0 +1,107 @@
+@import '/src/styles/common.scss';
+@import url('https://fonts.cdnfonts.com/css/proxima-nova-alt-lt-2');
```

Comment:
@aftabrehan i think we can use this font?

Need to put this import into common instead of here.

Reply:
@mrxshinji  Done!